### DISCUSSION TITLE
display of multidisc albums

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#358] [Q&A] [answered]
myMPD version: 6.8.0

Multidisc album are showing a album cover for each CD of the album. If accessing one CD of the multidisc album, all tracks are shown in the correct way. Is it possible to get only one cover shown in the album view for multidisc albums? The files are stored within one folder, CD number is tagged.

**Comment by jcorporation** [comment, marked as answer]
You must enable the Disc tag in the myMPD setttings.

**Comment by payback007** [comment]
Thank you very much, it was quite easy ;-)
